src/connectors/configure/connectConfigure.js

```diff
@@ -57,7 +57,7 @@ export default function connectConfigure(renderFn = noop, unmountFn = noop) {
         return searchParameters => {
           // merge new `searchParameters` with the ones set from other widgets
           const actualState = this.removeSearchParameters(helper.getState());
-          const nextSearchParameters = enhanceConfiguration({})(
+          const nextSearchParameters = enhanceConfiguration()(
```

Comment:
if this argument isn't used anymore, why isn't it simply one function?

Reply:
Yep indeed will drop the first level function.